demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss

```diff
@@ -73,3 +73,7 @@
 .font-size-0 {
     font-size: 0;
 }
+
+.font-size-1_2 {
```

Comment:
As we are soon moving those into yaml/tailwind, i do not recommend to add it here. Also, why do we need a "one off" font size?

Reply:
I was struggling with style overrides in the flyout component, this way appeared as simple solution to me with just minor drawbacks. However what would your proposal be how to handle this case?

Reply:
i see. in that case, could you move the css declaration into the project override?